eruditorg/templates/public/journal/partials/issue_detail_section.html

```diff
@@ -1,66 +1,17 @@
-  {% if section1.titles.main %}
-  <h3 class="section">
-    {{ section1.titles.main }}
-    {% for paral in section1.titles.paral %}/ {{ paral }}{% endfor %}
-  </h3>
+{% for group in articles_per_section.groups %}
+  {% if group.type == 'objects' %}
+    <ol class="issue_detail--articles unstyled section-{{ group.level }}">
+      {% for article in group.objects %}
+      {% include "public/journal/partials/issue_detail_article_item.html" %}
+      {% endfor %}
+    </ol>
+  {% else %}
+      <h{{ group.level|add:"1" }} class="section">
+      {{ group.titles.main }}
```

Comment:
Small typographic adjustment for multilingual section titles - make sure to display a non-breaking space *after* each title and before a slash. In Spanish, the title is stuck to the slash
See: http://localhost:8000/fr/revues/as/2011-v35-n3-as5007734/ ("Reseñas temáticas/")